content/en/docs/concepts/configuration/secret.md

```diff
@@ -167,7 +167,7 @@ However, using the [TokenRequest](/docs/reference/kubernetes-api/authentication-
 subresource to obtain a token to access the API is recommended instead.
 {{< /note >}}
 
-### Docker config Secrets
+### Docker Registry config Secrets
```

Comment:
This new title is misleading @Kartik494 

If the secret holds the contents of a `~/.dockercfg` then it's a Docker configuration, no matter what registry the Secret allows you to authenticate to.

Reply:
@sftim Please take a look at the discussion here 
https://github.com/kubernetes/website/issues/30945#issuecomment-1047038591

Reply:
I followed that up with https://github.com/kubernetes/website/issues/30945#issuecomment-1060524272

Reply:
Thanks for review on this PR @sftim i am looking forward for further Changes to be done in this PR

Reply:
@sftim Is there any other PR has been placed in main branch regarding changing in `Re title of docker config section`?